models/tracker/src/index.ts

```diff
@@ -1350,7 +1350,21 @@ export function createModel (builder: Builder): void {
   })
 
   builder.mixin(tracker.class.Issue, core.class.Class, view.mixin.ClassFilters, {
-    filters: ['space']
+    filters: [
+      'status',
+      'assignee',
+      'createdBy',
+      'priority',
+      'labels',
+      'title',
+      'sprint',
+      'component',
+      'dueDate',
+      'createOn',
+      'modifiedOn',
+      'modifiedBy'
+    ],
+    ignoreKeys: ['number', 'estimation', 'attachedTo', 'space']
```

Comment:
You don't need ignore hidden fields. Also 'space' filter here is good especially for 'my issues' mode

Reply:
There are no hidden fields I think. If I remove any key from this array, the filter appears in the filter list.
Thanks for note about "space" - removed.